apps/challenges/views.py

```diff
@@ -3330,6 +3336,19 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                             "config_id": data["config_id"],
                         },
                     )
+                elif (
+                    challenge_test_annotation_file
+                    and challenge_phase.annotations_uploaded_using_cli
```

Comment:
Same here. Did you test updating Phase meta fields after uploading annotations from cli?

Reply:
Since this a partial update it would update all the data given except the test annotation file preventing the overwriting.

Reply:
@savish28 the update in this `elif` block just has `partial=True` where are we excluding annotations from being updated?

Reply:
Can you share how you tested this change?

Reply:
![final_overwrite_annotatie](https://user-images.githubusercontent.com/32800267/111599671-8cde1000-87f6-11eb-9ee2-9142700ae61b.gif)
@Ram81 This is the basic example in which shows the following
- `annotations_uploaded_using_cli` False then description and test annotation both are overwritten
- `annotations_uploaded_using_cli` True then description is changed but test annotation is not overwritten as desired.